tests/test_templates.py

```diff
@@ -70,19 +74,80 @@ def test_multi_page_gui_template(tmpdir):
         # Assert post_gen_project hook is successful
         assert "taipy==" in requirements_file.read()
 
-    # Run the templates on a subprocess and get stdout and stderr after timeout
-    with subprocess.Popen(
-        [sys.executable, os.path.join(tmpdir, "foo_app", "main.py")], stdout=subprocess.PIPE, stderr=subprocess.PIPE
-    ) as proc:
-        try:
-            stdout, stderr = proc.communicate(timeout=5)
-        except subprocess.TimeoutExpired:
-            proc.kill()
-            stdout, stderr = proc.communicate()
+    stdout = _run_template(os.path.join(tmpdir, "foo_app", "main.py"))
 
-    # Print the eror if there is any (for debugging)
-    if stderr := str(stderr, "utf-8"):
-        print(stderr)
+    # Assert the message when the application is run successfully is in the stdout
+    assert "[Taipy][INFO]  * Server starting on" in str(stdout, "utf-8")
+
+
+def test_scenario_management_with_toml_config(tmpdir):
+    cookiecutter(
+        template="src/taipy/templates/scenario-management",
+        output_dir=tmpdir,
+        no_input=True,
+        extra_context={
+            "root folder name": "foo_app",
+            "main file": "main.py",
+            "application title": "bar",
+            "use toml config": "yes",
```

Comment:
```suggestion
            "root_folder_name": "foo_app",
            "main_file": "main.py",
            "application_title": "bar",
            "use_toml_config": "yes",
```

Reply:
Oops. I forgot to roll back this try. Thx!

By the way, I was wondering, @trgiangdo, if instead of exposing to the user the name of the cookiecutter variable (`root_folder_name`, for instance), we could expose another string instead, like a proper sentence in English (like "Your application root folder name").

Reply:
I will investigate this.